### ISSUE TITLE
Discuss about adding an API versionning.

### ISSUE THREAD

**Issue by Antoninbln** [ChauffeurPrive/nestor-api#66]

**Comment by maximejimenez** [comment]
@ChauffeurPrive/nestor-maintainers @Antoninbln 
I suggest we discuss this here

**Comment by maximejimenez** [comment]
My concern is that IMO it is not really high priority but maybe I am not seeing what is the use case yet for the versioning of the API.

I am not saying it will never be useful, I just don't want us to over-complicate the project now if we don't really need these kind of features.

**Comment by Kenny407** [comment]
I agree with Maxime. In your past experience with Nestor, did you notice the need of having API versioning that caused some pain maintainability-wise?
